### ISSUE TITLE
Hey need help

### ISSUE THREAD

**Issue by Qwiq973** [Arthurhoeke/Cloud9#1]
Hello @Arthurhoeke so I saw your issue post on SteamKit about your Windows Forrm is not connecting, I got the same problem. And you said that you fixed it and I wonder if you can help me out with that?

**Comment by Arthurhoeke** [comment]
I resolved the SteamKit not connecting problem by using an older version of SteamKit

**Comment by Qwiq973** [comment]
And you used the same code that you posted?

**Comment by Arthurhoeke** [comment]
No, the newer version of SteamKit has some changes to the API. You might want to go on Youtube and look at some tutorials with source code included, most of these tutorials are using the older version of SteamKit and you'll be able to use that code / use it as an example.
